src/FACT_internal.c

```diff
@@ -1143,21 +1145,22 @@ void FACT_INTERNAL_ActivateEvent(
 		/* Ramp/Equation */
 		if (evt->value.settings & 0x01)
 		{
-			/* FIXME: Incorporate 2nd derivative into the interpolated pitch */
+			/* FIXME: Incorporate 2nd derivative into the interpolated pitch (slopeDelta) */
```

Comment:
When the slopeDelta is incorporated into this it will change this whole section to a incremental algorithm instead of a lerp. It'll be faster too, not that it's a hot spot (not even I use even linear ramps at this point). It's nice to know it's there though!

Also, since the first svResult intermediate calculation is just the endValue it could be calculated up front, but we'll just have to change it all again when slopeDelta is used.